test/integration/targets/callback_default/runme.sh

```diff
@@ -119,6 +119,7 @@ export ANSIBLE_NOCOLOR=1
 # Default settings
 export ANSIBLE_DISPLAY_SKIPPED_HOSTS=1
 export ANSIBLE_DISPLAY_OK_HOSTS=1
+export ANSIBLE_DISPLAY_INCLUDES=1
```

Comment:
Since this is the default value, it's not necessary to set this variable on the existing test cases. In fact, this should _not_ be set to ensure it doesn't change the existing behavior.

Reply:
Please add an additional test case with `ANSIBLE_DISPLAY_INCLUDED_TASKS=0` to check the new behavior.